<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GridMap --emotionbot</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>497</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>搜索路径</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>选项</string>
    </property>
    <addaction name="action9_9_10"/>
    <addaction name="action16_16_40"/>
    <addaction name="action_16_30_99"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>清除路径</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action9_9_10">
   <property name="text">
    <string>初级 9*9 10个网格</string>
   </property>
  </action>
  <action name="action16_16_40">
   <property name="text">
    <string>中级 16*16 40个网格</string>
   </property>
  </action>
  <action name="action_16_30_99">
   <property name="text">
    <string>高级 16*30 99个网格</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
